packages/office-ui-fabric-react/src/components/Check/Check.base.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  classNamesFunction,
+  css,
+  customizable,
+} from '@uifabric/utilities';
+import {
+  ICheckProps,
+  ICheckStyleProps,
+  ICheckStyles
+} from './Check.types';
+import { Icon } from '../../Icon';
+import { getStyles } from './Check.styles';
+
+const getClassNames = classNamesFunction<ICheckStyleProps, ICheckStyles>();
+
+@customizable('Check', ['theme'])
+export class CheckBase extends BaseComponent<ICheckProps, {}> {
+  public static defaultProps: ICheckProps = {
+    checked: false
+  };
+
+  public shouldComponentUpdate(newProps: ICheckProps) {
+    return this.props.checked !== newProps.checked;
+  }
+
+  public render(): JSX.Element {
+    const {
+      checked,
+      className,
+      theme
+    } = this.props;
+
+    const classNames = getClassNames(getStyles!, { theme: theme!, className, checked });
+
+    return (
+      <div
+        className={ css(
```

Comment:
css doesn't need to be used and should be removed. Can you move all of the logic to getStyles so that we have it in one place?